scripts/key_placer.sh

```diff
@@ -74,9 +74,7 @@ for file_path in "${files[@]}"; do
 
   # When encrypting ensure the plaintext file exists and the encrypted file does not.
   if [[ $1 == "encrypt" ]]; then
-    if [[ -f "$encrypted_file_path" ]]; then
-        continue
-    elif [[ ! -f "$file_path" ]]; then
+    if [[ ! -f "$file_path" ]]; then
```

Comment:
I assume, that before we were overriding encrypted configs if they are already exists and now we will override them, correct? If yes - should we update the comment on line 75?

Reply:
Yes, that's the intended behavior :)
And I'll update the comment, good catch!